db/migrate/20150219194157_remove_location_from_stations.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveLocationFromStations < ActiveRecord::Migration
+  def change
+    remove_column :stations, :location, :string
```

Comment:
Is there any interesting information in the `location` column in production? I'm guessing there isn't, but if there *is* you'll want to do something with it before dropping the column.